haas/api.py

```diff
@@ -510,6 +510,25 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
+def list_networks():
+    """List all networks.
+    Returns a JSON dictionary of dictionaries, indexed by the name of the network.
+    The interior dictionaries are indexed by project and channel.
+    
+    Example:  {"netA": {"channel": "101", "project": "qproj-01"}}
+    """
+    db = model.Session()
+    networks = db.query(model.Network).all()
+    result = {}
+    for n in networks:
+        if n.access is not None:
+            net = {'channel': n.network_id, 'project': n.access.label}
```

Comment:
@lokI8 I forgot to mention that, Ravi and I've discussed on what channel should point to this afternoon and he would like it point to the network_id. Should I add the vlan number for it? 

Thanks